pie/utils/objects.py

```diff
@@ -96,3 +96,84 @@ def check(reaction, user):
                     with contextlib.suppress(nextcord.Forbidden):
                         await message.remove_reaction("▶️", user)
                     await message.edit(embed=self.pages[pagenum])
+
+
+class ConfirmView(nextcord.ui.View):
+    """Class for making confirmation embeds easy.
+    The right way of getting response is first calling wait() on instance,
+    then checking instance attribute `value`.
+    Attributes:
```

Comment:
It's nitpicking, but I'd like to see one empty line before every `Attributes:` and `Args:` line, so it is easier to read.